src/store/storage-controller.ts

```diff
@@ -80,26 +80,24 @@ export class StorageController {
       }
     }
 
-    await messageStore.put(tenant, message, indexes);
-    await eventLog.append(tenant, messageCid);
+    await this.messageStore.put(tenant, message, indexes);
+    if (this.eventLog) { await this.eventLog.append(tenant, messageCid); }
   }
 
-  public static async query(
-    messageStore: MessageStore,
-    dataStore: DataStore,
+  public async query(
```

Comment:
should we rename this `queryRecordsWrites` to better differentiate it from `queryMessages`?

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/42f36fb0385704079ff7f0a11a32f953d80a25c7